load_agent.py

```diff
@@ -50,6 +41,15 @@ def load_agent(env, directory="agent", gpu=0, epsilon=0.3):
     return agent
 
 
+def create_environment(imagefile='image_locations.txt', boxfile='bounding_boxes.npy', gpu=0):
+    locations = np.loadtxt(imagefile, dtype=str).tolist()
+    bboxes = np.load(boxfile)
+
+    env = TextLocEnv(locations, bboxes, gpu)
+
+    return env
```

Comment:
maybe emit variable here?

Reply:
This is not a change of yours it seems. Maybe change it anyways? :)

Reply:
👍 I'll change it.